reaktive/src/jsMain/kotlin/com/badoo/reaktive/scheduler/MainScheduler.kt

```diff
@@ -19,30 +19,30 @@ internal class MainScheduler : Scheduler {
 
         private var _isDisposed = false
 
-        private val timeoutIds = mutableListOf<Int>()
-        private val intervalIds = mutableListOf<Int>()
+        private val timeoutIds = mutableListOf<Long>()
```

Comment:
Not sure about changing to Long, but it made it easier to differentiate between 
`fun clearTimeout(interval: Long)` in `Timers` object and the external functions.